ProcessLib/Parameter/CurveScaledParameter.h

```diff
@@ -29,13 +29,24 @@ struct CurveScaledParameter final : public Parameter<T> {
     }
 
     bool isTimeDependent() const override { return true; }
+
+    void setParameter(ParameterBase const* parameter)
```

Comment:
This will be only used in the tests, right? Actually I think, we could live without this method.

Reply:
Yes, I added this for the testing, otherwise one would need to construct a vector of unique pointers to the parameters, s.t. the `initialize` function could be used.

Maybe moving the `findParameter` call and use the `setParameter` instead of `initialize` would result in better interface. The `initialize` function is implemented only in the CurveScaledParameter. (I guess the discussion is for another coffee ;)